src/style.rs

```diff
@@ -262,7 +274,7 @@ impl Style {
     /// assert_eq!(false, Style::default().bold().is_plain());
     /// ```
     pub fn is_plain(self) -> bool {
-        self == Style::default()
+        self == Style::new()
```

Comment:
We cannot constify this method since the equal operand is not usable in const context